src/commands/cacert/export.ts

```diff
@@ -36,22 +34,23 @@ export default class Export extends Command {
       env: 'CHE_CA_CERT_LOCATION',
       default: ''
     }),
+    [ACCESS_TOKEN_KEY]: accessToken,
```

Comment:
do we really need to provide access token?

Reply:
Yes, as we need to read the secret from Che namespace

Reply:
I don't understand why ^(

Reply:
Ah, yes, you are right. Just rechecked, it works without it. Thank you, will delete it.